Python/specialize.c

```diff
@@ -465,6 +466,21 @@ _PyCode_Quicken(PyCodeObject *code)
             }
             i += caches;
         }
+        else if (opcode == LOAD_CONST) {
+            /* We can't do this in the bytecode compiler as
+             * marshalling can intern strings and make them immortal. */
+            oparg = (oparg << 8) | instructions[i].op.arg;
```

Comment:
I'd put this above, before "if (caches)". Then below you don't need the same line again in case of `EXTENDED_ARG`, just `if (opcode != EXTENDED_ARG) oparg = 0;`.

This will also future proof this in case another case is added that needs the `oparg`.